CONTRIBUTING.md

```diff
@@ -34,7 +34,13 @@ The `phpunit.xml.dist` file is used as the default configuration file for the
 test suite. In addition to various PHPUnit options, it defines required
 `MONGODB_URI` and `MONGODB_DATABASE` environment variables. You may customize
 this configuration by creating your own `phpunit.xml` file based on the
-`phpunit.xml.dist` file we provide.
+`phpunit.xml.dist` file we provide. To run the tests in serverless mode, set the
+`MONGODB_IS_SERVERLESS` environment variable to `on`.
+
+To run tests against a cluster that requires authentication, either include the
+credentials in the connection string given in the `MONGODB_URI` environment
+variable, or set the `MONGODB_USERNAME` and `MONGODB_PASSWORD` environment
+variables accordingly.
```

Comment:
I assume if credentials are expressed in both forms that the username/password will take precedence given how array options override the connection string. Do you want to mention that here?

I don't think we need to go so far as raising an error in that case (it'd be annoying to have to inspect the connection string to detect this).

Reply:
I've added a note that env variables override URI credentials. Thanks for pointing it out, and I agree about not raising an error.